static/js/whyis_vue/components/table-view/components/Tableview.vue

```diff
@@ -0,0 +1,84 @@
+<template>
```

Comment:
TableView.vue? Fix case?

Reply:
renamed all components and files to camelCase